lib/stripe_mock/request_handlers/validators/param_validators.rb

```diff
@@ -110,9 +110,41 @@ def validate_create_plan_params(params)
 
       end
 
+      def validate_create_price_params(params)
+        price_id = params[:id].to_s
+        product_id = params[:product]
```

Comment:
Something worth noting, stripe-rails gem asks for the `product_id` instead of `product` when setting up the initializer.

references;
https://github.com/tansengming/stripe-rails (see prices.rb readme)
https://stripe.com/docs/api/prices/create#create_price-product

Reply:
Yeah, stripe-rails requires `product_id` and the Stripe API uses `product`.  I think that difference is because stripe-rails will optionally create a product and it has abstracted some of the Stripe API so that it makes the right request without the user having to set `product_data` in stripe-rails.

For this gem where it mocks the Stripe API I think `product` is the correct param to use.